ci/taos/checker-pr-audit-async.sh

```diff
@@ -286,6 +286,12 @@ do
     fi
 done
 
+if [[ ${build_test_fail} -eq 1 ]]; then
```

Comment:
Please use a capital character to figure out if the variable is "export ****" statement. 
```
s/build_test_fail/BUILD_TEST_FAIL/
```

Reply:
updated